import-scripts/fetch-dmp-data-for-import.sh

```diff
@@ -694,32 +694,38 @@ MY_FLOCK_FILEPATH="/data/portal-cron/cron-lock/fetch-dmp-data-for-import.lock"
     # CDM Fetch is optional -- does not break import if it fails, but will send notif
 
     echo "fetching CDM clinical demographics & timeline updates from S3..."
-    sh $PORTAL_HOME/scripts/pull-cdm-data.sh
+    download_from_s3 "$MSK_CHORD_DATA_HOME" "msk-chord" "cdm-deliverable"
 
     if [ $? -gt 0 ] ; then
         sendPreImportFailureMessageMskPipelineLogsSlack "S3 Failure: CDM data update"
     else
-        sh $PORTAL_HOME/scripts/merge-cdm-data.sh mskimpact
+        sh $PORTAL_HOME/scripts/merge-cdm-data.sh mskimpact $MSK_CHORD_DATA_HOME/mskimpact $MSK_IMPACT_DATA_HOME $MSK_IMPACT_DATA_HOME
         if [ $? -gt 0 ] ; then
             sendPreImportFailureMessageMskPipelineLogsSlack "Error: CDM merge for MSKIMPACT"
         fi
 
-        sh $PORTAL_HOME/scripts/merge-cdm-data.sh mskimpact_heme
+        sh $PORTAL_HOME/scripts/merge-cdm-data.sh mskimpact_heme $MSK_CHORD_DATA_HOME/mskimpact_heme $MSK_HEMEPACT_DATA_HOME $MSK_HEMEPACT_DATA_HOME
         if [ $? -gt 0 ] ; then
             sendPreImportFailureMessageMskPipelineLogsSlack "Error: CDM merge for HEMEPACT"
         fi
 
-        sh $PORTAL_HOME/scripts/merge-cdm-data.sh mskarcher
+        sh $PORTAL_HOME/scripts/merge-cdm-data.sh mskarcher $MSK_CHORD_DATA_HOME/mskarcher $MSK_ARCHER_UNFILTERED_DATA_HOME $MSK_ARCHER_UNFILTERED_DATA_HOME
         if [ $? -gt 0 ] ; then
             sendPreImportFailureMessageMskPipelineLogsSlack "Error: CDM merge for ARCHER"
         fi
 
-        sh $PORTAL_HOME/scripts/merge-cdm-data.sh mskaccess
+        sh $PORTAL_HOME/scripts/merge-cdm-data.sh mskaccess $MSK_CHORD_DATA_HOME/mskaccess $MSK_ACCESS_DATA_HOME $MSK_ACCESS_DATA_HOME
         if [ $? -gt 0 ] ; then
             sendPreImportFailureMessageMskPipelineLogsSlack "Error: CDM merge for ACCESS"
         fi
     fi
 
+    # Upload merged CDM cohorts to databricks
+    upload_to_s3 "$MSK_IMPACT_DATA_HOME" "mskimpact" "mskimpact-databricks"
```

Comment:
potentially uploading if we don't succeed merging cdm? Maybe move calls into the exist status check?